receiver/prometheusexecreceiver/testdata/end_to_end_metrics_test/test_prometheus_exporter windows.go

```diff
@@ -0,0 +1,50 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
This file appears to be a copy of test_prometheus_exporter.go. If that's the case instead of duplicating it export `server` func and use here.

Reply:
And actually, the space in the file name breaks `make checklicense` so I entirely removed this file and designed new unit tests to cover the issue that is being fixed here. Sorry for the repeated issues, I'm still struggling with the pipeline here... (issues that I discovery only once run as GitHub Actions and that should be caught earlier...)